tests/core/test_addasset.py

```diff
@@ -0,0 +1,74 @@
+import os
+from tempfile import TemporaryDirectory
+from unittest import TestCase
+
+import pystac
+
+from stactools.core import add_asset_to_item
+from tests import test_data
+from tests.utils import create_temp_copy
+
+
+class AddAssetTest(TestCase):
+    def test_add_asset_to_item(self):
+        with TemporaryDirectory() as tmp_dir:
+            item_path = create_temp_copy(
+                test_data.get_path("data-files/core/simple-item.json"),
+                tmp_dir,
+                "item.json",
+            )
+            item = pystac.read_file(item_path)
+            assert isinstance(item, pystac.Item)
+            asset = item.get_assets().get("test-asset")
+            assert asset is None
+            item_path = item.get_self_href()
+
+            asset_path = test_data.get_path("data-files/core/byte.tif")
+            asset = pystac.Asset(
+                asset_path, "test", "placeholder asset", roles=["thumbnail", "overview"]
+            )
+            item = add_asset_to_item(item, "test-asset", asset)
+
+            asset = item.get_assets().get("test-asset")
+            assert isinstance(asset, pystac.Asset), asset
+            assert asset.href is not None, asset.to_dict()
+            assert os.path.isfile(asset.href), asset.to_dict()
+            assert asset.title == "test", asset.to_dict()
+            assert asset.description == "placeholder asset", asset.to_dict()
+            self.assertListEqual(asset.roles, ["thumbnail", "overview"])
+
+    def add_asset_move(self):
```

Comment:
```suggestion
    def test_add_asset_move(self):
```

Otherwise, it isn't picked up as a test.

Reply:
oops, I totally overlooked this one.